src/game/GameStateMatch.cpp

```diff
@@ -13,15 +13,15 @@
 #include "../log/log.h"
 
 GameStateMatch::GameStateMatch(Game& g,  int gameWidth, int gameHeight) : GameState(g), player(),
-                               level(),  base(), camera(gameWidth,gameHeight){
+        level(),  base(), camera(gameWidth,gameHeight) {
```

Comment:
Do these empty initializers need to be here?
Can't you just leave them out and the compiler will automatically call them?